tests failing:
```
======================================================================
FAIL: custom.enikshay.tests.two_b_datamigration.test_logging:TestDRTBImportHistoryCommand.test_get_outcome
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/enikshay/tests/two_b_datamigration/test_logging.py", line 138, in test_get_outcome
    self.assertNotIn("Traceback", output, "Expected list of case ids, but output was: {}".format(output))
AssertionError: Expected list of case ids, but output was: Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py", line 1910, in handle
    import_format.constants, row
  File "/mnt/commcare-hq-ro/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py", line 536, in get_case_structures_from_row
    person_case_properties = get_person_case_properties(domain, column_mapping, row)
  File "/mnt/commcare-hq-ro/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py", line 658, in get_person_case_properties
    properties.update(get_disease_site_properties_for_person(column_mapping, row))
  File "/mnt/commcare-hq-ro/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py", line 875, in get_disease_site_properties_for_person
    props = get_disease_site_properties(column_mapping, row)
  File "/mnt/commcare-hq-ro/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py", line 834, in get_disease_site_properties
    value = xlsx_value.replace('EP ', 'extra pulmonary ').\
AttributeError: 'NoneType' object has no attribute 'replace'
```
and 
```
FAIL: custom.enikshay.tests.two_b_datamigration.test_logging:TestLogCreation.test_simple_import
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/enikshay/tests/two_b_datamigration/test_logging.py", line 108, in test_simple_import
    5 + len(ALL_DRUGS)
AssertionError: 1 != 27
```

@sheelio

--- a/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py
+++ b/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py
@@ -1527,8 +1527,8 @@ def clean_weight_band(value):
         "Less than 16": "drtb_conventional_lt_16",
         "16-29": "drtb_conventional_16_29",
         "30-45": "drtb_conventional_30_45",
-        "16-25": "drtb_old_conventional_16_25",
-        "26-45": "drtb_old_conventional_26_45",
+        "16-25": "drtb_conventional_old_16_25",
+        "26-45": "drtb_conventional_old_26_45",
         "46-70": "drtb_conventional_46_70",
         "Above 70": "drtb_conventional_gt70"
     }[value]

--- a/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py
+++ b/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py
@@ -681,7 +681,6 @@ def get_occurrence_case_properties(column_mapping, row):
         "current_episode_type": "confirmed_drtb",
         "initial_home_visit_status": "completed" if initial_visit_date else None,
         "initial_home_visit_date": clean_date(initial_visit_date),
-        "drtb_type": clean_drtb_type(column_mapping.get_value("drtb_type", row)),
         'name': 'Occurrence #1',
         'occurrence_episode_count': 1,
     }

--- a/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py
+++ b/custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py
@@ -334,7 +334,6 @@ MUMBAI_MAP = {
     "Am": 82,
     "dst_result_date": 83,
     "treatment_initiation_date": 89,
-    "drtb_type": 92,
     "mumbai_treatment_status": 92,
     "treatment_regimen": 93,
     "ip_to_cp_date": 95,
@@ -831,14 +830,15 @@ def get_diagnosis_properties(column_mapping, domain, row):
 
 def get_disease_site_properties(column_mapping, row):
     xlsx_value = column_mapping.get_value("site_of_disease", row)
+    if not xlsx_value:
+        return {}
+
     value = xlsx_value.replace('EP ', 'extra pulmonary ').\
         lower().\
         replace('extra pulmonary', 'extra_pulmonary').\
         replace('lymph node', 'lymph_node').\
         replace('pleural effusion', 'pleural_effusion')
 
-    if not value:
-        return {}
     if (not re.match("^extra_pulmonary \(other - .*$", value)
         and value not in [
         "pulmonary",
@@ -1316,6 +1316,9 @@ def clean_mumbai_treatment_status(value, treatment_initiation_date):
 
 
 def clean_patient_type(value):
+    if not value:
+        return None
+
     clean_value = value.lower().replace(' ', '_')
     if clean_value not in [
         "new",

